{"uid":"3a6eaae80e3d7e86","name":"Successful form submission [Personal-personal@test.com-Personal test message-\\u041b\\u0438\\u0447\\u043d\\u044b\\u0439]","fullName":"features/main_page.feature:Successful form submission","historyId":"78fb15ab50a486870631bc81b5acc5e3","time":{"start":1731553692856,"stop":1731553701248,"duration":8392},"status":"failed","statusMessage":"AssertionError: Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".form-submission-text\")","statusTrace":"fixturefunc = <function step_verify_success_message at 0x7fef79c75040>\nrequest = <FixtureRequest for <Function test_successful_form_submission[Personal-personal@test.com-Personal test message-\\u041b\\u0438\\u0447\\u043d\\u044b\\u0439]>>\nkwargs = {'main_page': <pages.main_page.MainPage object at 0x7fef7acea4c0>, 'message': 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n/opt/hostedtoolcache/Python/3.9.20/x64/lib/python3.9/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nmain_page = <pages.main_page.MainPage object at 0x7fef7acea4c0>\nmessage = 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\n\n    @then(cfparse('I should see the success message \"{message}\"'))\n    def step_verify_success_message(main_page: MainPage, message):\n>       expect(main_page.success_message).to_have_text(message)\nE       AssertionError: Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nE       Actual value: <element(s) not found> \nE       Call log:\nE       LocatorAssertions.to_have_text with timeout 5000ms\nE         - waiting for locator(\".form-submission-text\")\n\ntests/step_defs/test_main_page_bdd.py:30: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".form-submission-text\")","statusTrace":"fixturefunc = <function step_verify_success_message at 0x7fef79c75040>\nrequest = <FixtureRequest for <Function test_successful_form_submission[Personal-personal@test.com-Personal test message-\\u041b\\u0438\\u0447\\u043d\\u044b\\u0439]>>\nkwargs = {'main_page': <pages.main_page.MainPage object at 0x7fef7acea4c0>, 'message': 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n/opt/hostedtoolcache/Python/3.9.20/x64/lib/python3.9/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nmain_page = <pages.main_page.MainPage object at 0x7fef7acea4c0>\nmessage = 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\n\n    @then(cfparse('I should see the success message \"{message}\"'))\n    def step_verify_success_message(main_page: MainPage, message):\n>       expect(main_page.success_message).to_have_text(message)\nE       AssertionError: Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nE       Actual value: <element(s) not found> \nE       Call log:\nE       LocatorAssertions.to_have_text with timeout 5000ms\nE         - waiting for locator(\".form-submission-text\")\n\ntests/step_defs/test_main_page_bdd.py:30: AssertionError","steps":[{"name":"Given I open the main page","time":{"start":1731553692857,"stop":1731553695397,"duration":2540},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"When I fill the form with name \"Personal\", email \"personal@test.com\", message \"Personal test message\", and request type \"Личный\"","time":{"start":1731553695398,"stop":1731553695752,"duration":354},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"And I submit the form","time":{"start":1731553695753,"stop":1731553696242,"duration":489},"status":"passed","steps":[],"attachments":[{"uid":"1cb1bdcc24be108e","name":"Submission Response","source":"1cb1bdcc24be108e.txt","type":"text/plain","size":58}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true},{"name":"Then I should see the success message \"Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.\"","time":{"start":1731553696243,"stop":1731553701248,"duration":5005},"status":"failed","statusMessage":"Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".form-submission-text\")\n","statusTrace":"AssertionError: Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".form-submission-text\")\n\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":1,"hasContent":true},"afterStages":[],"labels":[{"name":"host","value":"fv-az1024-569"},{"name":"thread","value":"3048-MainThread"},{"name":"framework","value":"pytest-bdd"},{"name":"language","value":"cpython3"},{"name":"feature","value":"Main page form submission"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"personal@test.com"},{"name":"message","value":"Personal test message"},{"name":"name","value":"Personal"},{"name":"request_type","value":"Личный"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"f7c26f5ee8ed8c5c","reportUrl":"https://efimovnik.github.io/Kolyaefimov-test-automation/6//#testresult/f7c26f5ee8ed8c5c","status":"failed","statusDetails":"AssertionError: Locator expected to have text 'Спасибо! Ваш запрос уже получен.Я свяжусь с вами по указанному email в течение 3 дней или ранее, если я смогу что-то предложить в ответ.'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".form-submission-text\")","time":{"start":1731553268656,"stop":1731553276971,"duration":8315}}]},"tags":[]},"source":"3a6eaae80e3d7e86.json","parameterValues":["personal@test.com","Personal test message","Personal","Личный"]}